test/Transforms/test-listener-cse.mlir

```diff
@@ -0,0 +1,246 @@
+// RUN: mlir-proto-opt -allow-unregistered-dialect %s -pass-pipeline='builtin.func(test-listener-cse)' | FileCheck %s
+
```

Comment:
Same thing here re copied from ?
Maybe just a top of the file comment that this is fully copied from core and to not modify?